roles/organizr/tasks/main.yml

```
@@ -64,11 +64,13 @@
 - name: Create and start container
   docker_container:
     name: organizr
-    image: "organizrtools/organizr-v2:plex"
+    image: "organizr/organizr"
     pull: yes
     published_ports:
       - "127.0.0.1:7421:80"
     env:
+      fpm: "{{ (organizr.fpm|default(false,true)) }}"
```

```suggestion
      fpm: "{{ (organizr.fpm|default(false,true)) }}"
```
I think the default is causing this to fail and is likely not necessary as default values are populated.